my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
```

Comment:
You do not have any code checking the body to make sure it is correct. You need to do that and send a 400 response if there are problems. This is the case for all of your endpoints.